src/java/org/apache/cassandra/service/accord/AccordFetchCoordinator.java

```diff
@@ -430,10 +444,12 @@ protected synchronized void onReadOk(Node.Id from, CommandStore commandStore, Da
         streamData.streams.forEach((range, streamInfo) -> {
             if (streamInfo.hasData)
             {
+                logger.info("StreamPlan {} created for bootstrap of {} from {}", streamInfo.planId, range, from);
                 stream(streamInfo.planId).rangeReceived(range, from);
             }
             else
             {
+                logger.info("StreamPlan {} created for bootstrap of {} from {} with no data to stream; succeeding immediately.", streamInfo.planId, range, from);
```

Comment:
just to make sure: was this meant to be `info`?

Reply:
I was thinking `warn` just because it's an error, but `info` works for me if this was intended.